rnd/autogpt_server/autogpt_server/blocks/discordsendmessage.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from autogpt_server.data.block import Block, BlockSchema, BlockOutput
+
+
+class discordsendmessage(Block):
+    class Input(BlockSchema):
+        webhook_url: str
+        message: str
+
+    class Output(BlockSchema):
+        status: str
+
+    def __init__(self):
+        super().__init__(
+            id="b3a9c1f2-5d4e-47b3-9c4e-2b6e4d2c4f3e",
+            input_schema=discordsendmessage.Input,
+            output_schema=discordsendmessage.Output,
+            test_input={
+                "webhook_url": "https://discord.com/api/webhooks/your_webhook_url",
```

Comment:
Are there any credentials required?

Reply:
No credentials are required, its all part of the webhook url